src/common/test-util/src/temp_dir.rs

```diff
@@ -0,0 +1,19 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use tempfile::TempDir;
```

Comment:
You could pub use the TempDir
```suggestion
pub use tempfile::TempDir;
```

Then other crates don't need to import tempfile directly. They can import TempDir from this crate.
```rust
use common_test_util::temp_dir::TempDir;
```

Reply:
oh.. nice one! it's a shame that I've overlooked it